### ISSUE TITLE
Cannot use variables in queres

### ISSUE THREAD

**Issue by AlexSemenovIRL** [AquaQAnalytics/grafana-kdb-datasource-ws#4]
Hi Guys, 

Really like this version of the plugin,  is it possible to use variables in the queries, particularly nested variables, e.g. venues, products, symbols...

Thanks

**Comment by CallumOrmerod** [comment]
Hello Alex, Are you referring to the Grafana variables?  If so, then at the moment this adaptor doesn't support them, but it's something we're hoping to add in the next version.

**Comment by AlexSemenovIRL** [comment]
Yes, those will be very useful to have. Do you have a potential eta?

**Comment by CallumOrmerod** [comment]
Not yet I'm afraid.  Although support for the Grafana variables is definitely one of things on our radar we were going to see what feedback we got from our initial release to help inform and prioritise the to-do's.  I can keep you posted though.

**Comment by dalmatine** [comment]
I'd be very interested to see support for variables in kdb queries.

**Comment by dmitry-ee** [comment]
upvote to this request

**Comment by shikanot** [comment]
Hi @AlexSemenovIRL 
Thank you for opening this issue. I'm really interested in to use Grafana variables. Our team is using this plugin and would like to have this feature. Could you tell me when you are planning to do this IF you have a chance?
